src/OrleansTestingHost/StorageTestConstants.cs

```diff
@@ -78,6 +78,19 @@ static StorageTestConstants()
             DataConnectionString = DEFAULT_STORAGE_DATA_CONNECTION_STRING;
         }
 
+        public static bool UsingAzureLocalStorageEmulator
+        {
+            get
+            {
+                string connString = DataConnectionString;
+                bool usingLocalWAS = DEFAULT_STORAGE_DATA_CONNECTION_STRING.Equals(connString, StringComparison.OrdinalIgnoreCase);
+                string msg = string.Format("Using Azure local storage emulator = {0}", usingLocalWAS);
+                Console.WriteLine(msg);
+                Trace.WriteLine(msg);
+                return usingLocalWAS;
```

Comment:
Sure. My guess is he added these side effects in order to debug the testing environment. I am quite sure we can comment them out.